src/types/aria-query.d.ts

```diff
@@ -0,0 +1,395 @@
+// Adapted from flow types here: https://github.com/A11yance/aria-query/blob/main/flow/aria.js
+declare module 'aria-query' {
```

Comment:
The available types for `aria-query` were outdated and had some key things missing, so I copied and adapted these from their Flow types.